src/utils/functions/getLocalizedResultStrings.ts

```diff
@@ -37,7 +37,7 @@ export const getLocalizedResultString = (
       return t('No Response Yet');
 
     case DisplayResultEnum.AppointmentResultNotInterested:
-      return t('Does not want to meet');
+      return t('Not Interested');
```

Comment:
Good catch

Reply:
Was this missed when we did Task Phases?

Reply:
Yeah, we missed it. In the future, we should get it from the API.